chunkator/__init__.py

```diff
@@ -11,9 +11,12 @@ def chunkator(queryset, chunk_size):
     length. We're saving at least a ``count()`` query on QuerySets, or a
     CPU-and-RAM-consuming ``len(queryset)`` query.
     """
-    pk = 0
+    pk = None
     while True:
-        page = queryset.filter(pk__gt=pk).order_by('pk')[:chunk_size]
+        queryset = queryset.order_by('pk')
```

Comment:
If you want to order on a special fields, so use another pagination tool as django-pagination or the legacy Paginator, and if the order does not matter use django-chunkator if performance matters.